### PR TITLE
MCS-1813 Updated the webenabled app to be packageable with pyinstaller

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Note the new run command.

### CODE REVIEW

File: webenabled/mcsweb.py [hunk 1]
```diff
@@ -154,3 +175,7 @@ def handle_scene_selection():
         step_output=step_output
     )
     return resp
+
+
+if __name__ == "__main__":
+    app.run(host='0.0.0.0', port=8080, debug=True)
```
Comment (ThomasSchellenbergNextCentury) [inline webenabled/mcsweb.py:181]: Note that you now run this with just `python mcsweb.py` (we'll probably want to add some command line arguments to configure these settings in the future).

File: webenabled/README.md [hunk 2]
```diff
@@ -58,3 +48,29 @@ http://<machine.ip.address>:8080/mcs
...
+
+```
+./dist/mcsweb/mcsweb
+```
+
+Cleanup (do before you need to rebuild):
+
+```
+rm -rf build/ dist/
+```
```
Comment (ThomasSchellenbergNextCentury) [inline webenabled/README.md:76]: These are the instructions for building and running the package. I'd appreciate if someone could verify it works for them as well. We'll want to be able to run it on Windows and Mac.

**Comment by rartiss55** [comment]
Going to look at this closer tonight/tomorrow, but wanted to confirm that the setup instructions work for me on a Mac!

File: webenabled/mcsweb.py (hunk 1 above)
Comment (rartiss55) [reply webenabled/mcsweb.py:184]: Made a ticket here: https://nextcentury.atlassian.net/browse/MCS-1930

**Review by rartiss55 — APPROVED** [review]
Works + code LGTM.

**Comment by ThomasSchellenbergNextCentury** [comment]
I think I'll go ahead and merge this in, since it's be open for a while, and I'll check the Windows support once we get all of those tickets done (Jenkins deployment, auto-download). Does that sound good?

**Comment by phile-caci** [comment]
> I think I'll go ahead and merge this in, since it's be open for a while, and I'll check the Windows support once we get all of those tickets done (Jenkins deployment, auto-download). Does that sound good?

Sounds good.  My only note above was just instructions for installing pyinstaller on windows is a little different, but happy to kick this down the road to get this merged.

Via Powershell:
```
pyinstaller --add-data "templates;templates" --add-data "static;static" --add-data "scenes;scenes" --console mcsweb.py --log-level=DEBUG -y *>&1 | Tee-Object -Append -FilePath pyinstaller.out
```

File: webenabled/README.md [hunk 3]
```diff
@@ -58,3 +48,29 @@ http://<machine.ip.address>:8080/mcs
...
+
+```
+pip install -U pyinstaller
+```
+
+Build:
+
+```
+pyinstaller --add-data 'templates:templates' --add-data 'static:static' --add-data 'scenes:scenes' --console mcsweb.py --log-level=DEBUG &> pyinstaller.out
+```
```
[the remark this answers is not in the document]
Comment (phile-caci) [reply webenabled/README.md:64]: yup..l old comment now, never submitted.  Sry about that.

**Review by phile-caci — APPROVED** [review]
LGTM!

**Pull request merged**